core/subscription/subscriber.hpp

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_SUBSCRIBER_HPP
```

Comment:
Er, ma
```suggestion
#ifndef KAGOME_SUBSCRIPTION_SUBSCRIBER_HPP
```
Short header guards make me anxious.